tests/config/run_test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+##set -xe
```

Comment:
```suggestion
set -xe
```
It will help understand what is happening in the CI.

Reply:
just uploaded a new one. Thanks!